src/tests/ftest/launch.py

```diff
@@ -1140,6 +1140,31 @@ def archive_config_files(avocado_logs_dir, args):
         status = 16
     return status
 
+def archive_cov_usrlib_logs(avocado_logs_dir, args):
+    """Archive daos cov files to the avocado results directory.
+    Args:
+        avocado_logs_dir (str): path to the avocado log files
+        args (argparse.Namespace): command line arguments for this program
+    Returns:
+        int: status code.
+    """
+    # Create a subdirectory in the avocado logs directory for this test
+    destination = os.path.join(avocado_logs_dir, "daos_covs_usrlib")
+
+    # Copy any DAOS covs created on all hosts
+    hosts = list(args.test_servers)
+    hosts.append(socket.gethostname().split(".")[0])
+    print("Archiving host covs from {} in {}".format(hosts, destination))
+
+    # Copy any log files written to the DAOS_TEST_LOG_DIR directory
+    task = archive_files(destination, hosts,
+                         "/usr/lib/daos/TESTING/ftest/test.cov*",False, args)
```

Comment:
Nothing should be getting written into `/usr/lib/*`.  That namespace if for the system and could very well be on a read-only `/usr` filesystem.